host.go

```diff
@@ -78,7 +80,7 @@ func LoadHost(name string, storePath string) (*Host, error) {
 
 func ValidateHostName(name string) (string, error) {
```

Comment:
This seems like we could get some easy tests added for this?  Would you mind? :)